src/content/docs/build/reference/sdk/index.mdx

```diff
@@ -148,5 +151,12 @@ func main() {
 </Fragment>
 </LangTabs>
 
+## Authentication
+
+Authentication is managed for you when developing locally with the `space dev` command, 
+when using the SDK inside a [Micro](/docs/en/build/fundamentals/the-space-runtime/micros), 
+or when using the SDK in a Space app running in the browser (the SDK will not authenticate on [public micros or public routes](/docs/en/build/fundamentals/the-space-runtime/authentication#public-micros-and-routes)).
+
+Alternatively, you can authenticate with the Deta SDK by providing a [Data Key](/docs/en/build/fundamentals/data-storage#data-keys) in an environment variable called `DETA_PROJECT_KEY`.
```

Comment:
Maybe it should be noted that you can also authenticate by providing the Data/Project key to Base or Drive respectively even if you're not in an runtime that has the `environment variables` abstraction? You can opt-out from Space based auto-authentication to fetch/modify your data outside of Space too.

Reply:
yes we can / should do that